src/components/Sections/Contact.svelte

```diff
@@ -5,9 +5,9 @@
   import GetNotified from '../GetNotified/GetNotified.svelte';
 </script>
 
-<SectionWrapper id="contact" classes="text-white bg-navy">
-  <SectionContent classes="freq-container">
-    <SectionParagraph classes="">Image</SectionParagraph>
+<SectionWrapper id="contact" class="bg-navy text-white">
```

Comment:
order changed because prettier is now being applied to the tailwind where it wasn't before.